extensions/amp-social-share/1.0/social-share.jss.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createUseStyles} from 'react-jss';
+
+const focus = {
+  '&:focus': {
+    outline: '#0389ff solid 2px',
+    outlineOffset: '2px',
```

Comment:
Could you clarify what's this for? Not very familiar with the offset.

Reply:
This increases or decreases the distance from the edge of the element to the start of the outline.  (see above) :)